IHP/FileStorage/ControllerFunctions.hs

```diff
@@ -252,9 +253,21 @@ uploadToStorage :: forall (fieldName :: Symbol) context record (tableName :: Sym
     ) => Proxy fieldName -> record -> IO record
 uploadToStorage field record = uploadToStorageWithOptions def field record
 
+-- | Permanently removes a previously stored file from storage.
+removeFile :: (?context :: context, ConfigProvider context) => StoredFile -> IO (Either MinioErr ())
+removeFile StoredFile { path, url } = do
```

Comment:
Could we rename this to `removeFileFromStorage`? Otherwise the function will collide with `System.Directory.removeFile` which sometimes is in scope